students/BrianNisonger/Lesson3/assignment/basic_operations.py

```diff
@@ -0,0 +1,66 @@
+from peewee import *
+from customer_model import Customer
+from customer_model import database
+
+
+def model_to_dict(model):
+    model_dict = {
+        'customer_id': model.customer_id,
+        'name': model.first_name,
+        'last_name': model.last_name,
+        'home_address': model.home_address,
+        'phone_number': model.phone_number,
+        'email_address': model.email_address,
+        'status': model.status,
+        'credit_limit': model.credit_limit
+    }
+    return model_dict
+
+
+def add_customer(customer_id, name, last_name, home_address, phone_number,
+                 email_address, status, credit_limit):
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        raise
+    if not database.table_exists('Customer'):
+        database.create_tables([Customer])
```

Comment:
This is a clever idea, but it assumes you always start with an add. Which isn't terrible, if you do something else, you get an error and realize there is no table, but it isn't necessarily clear to the user that they should then just try to add a customer and the table will just magically appear. And, once there is a UI, you really just want to create the table when you start the ui. So, I'd have a special create table function.